src/main/java/com/puppycrawl/tools/checkstyle/JavadocDetailNodeParser.java

```diff
@@ -413,7 +413,19 @@ private static int getTokenType(ParseTree node) {
         final int tokenType;
 
         if (node.getChildCount() == 0) {
-            tokenType = ((TerminalNode) node).getSymbol().getType();
+            final int antlrTokenType = ((TerminalNode) node).getSymbol().getType();
+            //JavadocTokenTypes.HTML_TAG_NAME != JavadocParser.HTML_TAG_NAME.
+            //JavadocTokenTypes.EMBED_HTML_TAG_NAME != JavadocParser.EMBED_HTML_TAG_NAME
+            // See Issue #5139.
```

Comment:
I feel we need a better description here instead of `a != A, see issue`.
It is not clear in the issue why this remains.

Reply:
Updated.